lib/ansible.sh

```diff
@@ -16,7 +16,7 @@ strap::lib::import exec || . exec.sh
 
 STRAP_HOME="${STRAP_HOME:-}"; [[ -n "$STRAP_HOME" ]] || { echo "STRAP_HOME is not set" >&2; exit 1; }
 STRAP_USER_HOME="${STRAP_USER_HOME:-}"; [[ -n "$STRAP_USER_HOME" ]] || { echo "STRAP_USER_HOME is not set" >&2; exit 1; }
-STRAP_INTERACTIVE="${STRAP_INTERACTIVE:-}"; [[ -n "$STRAP_INTERACTIVE" ]] || STRAP_INTERACTIVE=true
+STRAP_SUDO_PROMPT="${STRAP_SUDO_PROMPT:-}"; [[ -n "$STRAP_SUDO_PROMPT" ]] || STRAP_SUDO_PROMPT=true
```

Comment:
So one backwards compat option could be to change this to:
```
STRAP_SUDO_PROMPT="${STRAP_SUDO_PROMPT:-$STRAP_INTERACTIVE}"; [[ -n "$STRAP_SUDO_PROMPT" ]] || STRAP_SUDO_PROMPT=true
```
Happy to make that change if folks think it would be useful.

Reply:
That change looks better from a backwards-compatibility stance IMO

Reply:
👍